tests/stress_tests/scripts/run_memcheck.py

```diff
@@ -66,6 +68,29 @@ def json_load(path_or_string):
         return json.loads(path_or_string)
 
 
+def get_cli_arg(key: str, args: list):
+    """ Get value of command-line argument
+    """
+    if key[0] != "-":
+        key = "-" + key
+    if key[-1] == "=":
+        key = key[0:-1]
+    val = ""
+    for i, arg in enumerate(args):
+        if key in arg:
+            if key == arg or "-" + key == arg:
```

Comment:
if `key` is full name OR clause would not be required

Reply:
Because implementation becomes difficult, I replaced `get_cli_arg` with additional ArgumentParser with `--test_conf` only